modules/cluster-logging-collector-log-forward-logs-from-application-pods.adoc

```diff
@@ -19,37 +19,38 @@ To specify the pod labels, you use one or more `matchLabels` key-value pairs. If
 .Example `ClusterLogForwarder` CR YAML file
 [source,yaml]
 ----
-apiVersion: logging.openshift.io/v1
+apiVersion: observability.openshift.io/v1
 kind: ClusterLogForwarder
 metadata:
-  name: <log_forwarder_name> <1>
-  namespace: <log_forwarder_namespace> <2>
+  name: <log_forwarder_name>
+  namespace: <log_forwarder_namespace>
 spec:
+  managementState: Managed
   pipelines:
-    - inputRefs: [ myAppLogData ] <3>
-      outputRefs: [ default ] <4>
-  inputs: <5>
+    - inputRefs: [ myAppLogData ] <1>
+      outputRefs: [ <output_type> ] <2>
+  inputs: <3>
     - name: myAppLogData
       application:
         selector:
-          matchLabels: <6>
+          matchLabels: <4>
             environment: production
             app: nginx
-        namespaces: <7>
+        namespaces: <5>
```

Comment:
This has changed in v6 to require `includes` or `excludes`    [example](https://github.com/openshift/cluster-logging-operator/blob/master/docs/administration/upgrade/v6.0_changes.adoc#inputs-specifications)